File: examples-sdk/typescript/035-basic-vil-service-module.ts
#!/usr/bin/env tsx
// 035-basic-vil-service-module — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 035-basic-vil-service-module.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();

File: examples-sdk/python/035-basic-vil-service-module.py
#!/usr/bin/env python3
"""035-basic-vil-service-module — Python SDK equivalent
Compile: vil compile --from python --input 035-basic-vil-service-module.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()
